src/types/on-call/index.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Date range type
+ */
+export type DateRange = {
+  start_date: string,
+  end_date: string
+};
+
+/**
+ * Access token
+ */
+export interface AccessToken {
+  created: Date;
+  access_token: string;
+  email?: string;
+  expires_in?: number;
+  refresh_token?: string;
+  refresh_expires_in?: number;
+  firstName?: string;
+  lastName?: string;
+  userId?: string;
+  roles?: string[];
+}
+
+/**
+ * Application configuration
+ */
+export interface Configuration {
+  auth: {
+    issuer: string;
+  },
+  api: {
+    apiKey: string;
+  },
+  onCall: {
+    bucketName: string;
+  },
+  pipedriveApi: {
+    apiKey: string,
+    apiUrl: string
+  },
+  splunkApi: {
+    apiKey: string
+    apiId: string,
+    teamOnCallUrl: string,
+    schedulePolicyName: string
+  }
+}
```

Comment:
As this isn't just used by the oncall lambdas, it should also be in the general types/index.ts